playbooks/robusta_playbooks/popeye.py

```diff
@@ -129,16 +131,17 @@ def popeye_scan(event: ExecutionBaseEvent, params: PopeyeParams):
     Displays a popeye scan report.
     """
 
+    sanitize_args = " ".join([f"'{arg}'" for arg in shlex.split(params.args)])
```

Comment:
@arikalon1  had to change it to this after some extra tests

Reply:
Yea ill do a quick check tomorrow , it might be better to change it to **shlex.join**. and replace my **f"'{arg}'"** with the shlex.quote.

Reply:
If you will, could you also do the same for krr playbook? It has similar code there

Reply:
yea np